package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "I used what we’ve learned and take a chance to continue practicing DOM manipulation by dynamically rendering a simple restaurant homepage! At the end, we use use JavaScript alone to generate the entire contents of the website!",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Haroonabdulrazaq/Restaurant-page.git"
+  },
+  "author": "Haroon",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Haroonabdulrazaq/Restaurant-page/issues"
+  },
+  "homepage": "https://github.com/Haroonabdulrazaq/Restaurant-page",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^5.0.1",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "file-loader": "^6.2.0",
+    "style-loader": "^2.0.0",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.4.0",
+    "webpack-cli": "^4.2.0"
+  },
```

Comment:
- [ ] As it was pointed out in the previous review, `webpack-dev-server` it's not added to your dependencies repository. Because of that, the `start` script it's not working right now. We need to either delete the `start` script or add `webpack-dev-server` to your dev dependencies. **Please we need to make this change before merging the branch**.